benchmarks/redis/run.sh

```diff
@@ -125,15 +125,17 @@ done
 rm -rf results
 mkdir -p results
 
-docker run --name redis-docker -d --privileged $MAXMEM $MAXSWAP $CPUSETS_CPU $CPUSETS_MEM -p 7551:7551 --entrypoint="/root/entrypoint.sh" $CONTAINER "$MEMPOLICY"
+docker run --name redis-docker -d --privileged $MEMPOLICY $MAXMEM $MAXSWAP $CPUSETS_CPU $CPUSETS_MEM --network=host $CONTAINER 
```

Comment:
Ideally, we would like to verify this command succeeds before proceeding and handle any errors appropriately.